util/chplenv/chpl_llvm.py

```diff
@@ -65,7 +65,7 @@ def has_compatible_installed_llvm():
 def get():
     llvm_val = overrides.get('CHPL_LLVM')
     if not llvm_val:
-        llvm_val = 'none'
+        llvm_val = 'unset'
```

Comment:
@daviditen / @mppf: I was hoping to suggest that, as a result of this PR, we could remove the explicit `CHPL_LLVM=none` settings on the linux32 test configurations (and maybe some others?).  But looking at the logic that follows this line, I don't think we can yet because things like `linux32` will fall through (as before), and end up with `unset` as their default value; whereas I think we want cases where `compatible_platform_for_llvm_default()` returns `false` to infer `CHPL_LLVM` to `none` (right?).

Reply:
Right. I think it'd be reasonable for linux32 to default to "none". I'm looking at making some other adjustments here and will fold that in to my effort.